### ISSUE TITLE
[Feature Request]: Add a button to bypass ( in the node's top right corner ) ( shown on mouseover ) 

### ISSUE THREAD

**Issue by JorgeR81** [Comfy-Org/ComfyUI_frontend#1126]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

I think bypassing is a feature so important that it deserves a button in each node.

### Proposed workflow

The button could be in the node's top right corner.
And it could be shown only when we mouseover this area.

Ideally this would be a node property that we could enable per node type. 
For instance, I would like to bypass a `Load Lora` or `Apply ControlNet`, but not a `Load Checkpoint` or `Load ControlNet`  node. 

### Additional information

My current workaround is placing each node in a group of its own, and enabling an rgthree setting, to add bypass buttons to all groups.

I've also made a feature request there to have buttons on the nodes ( https://github.com/rgthree/rgthree-comfy/issues/204 ),  a few months ago.
Other users were also interested, but we had no feedback from rgthree on this. 
He seems to be even more busy now, so I don't think this is going to be added anytime soon.

So it only works for groups for now :

![set](https://github.com/user-attachments/assets/72d42582-40e5-4fd7-9c26-715d7cd8fad7)

![loras](https://github.com/user-attachments/assets/5588ffc3-7e42-410f-b9bf-c783ddb0204b)

**Comment by JorgeR81** [comment]
If we can have a button on each widget ( https://github.com/Comfy-Org/ComfyUI_frontend/pull/994 ), I figured that we could also have one in each node.

**Comment by tiptinker** [comment]
You can select the node and Ctrl+B

**Comment by JorgeR81** [comment]
> You can select the node and Ctrl+B

Yeah I know, but once I started using the rgthree toggle in groups, I really liked this method.

**Comment by LukeG89** [comment]
In the meanwhile, another good alternative is to simplify the shortcuts like I did:

- `C` -> Collapse
- `B` -> Bypass
- `M` -> Mute

Obviously for the last one I had to replace Model Library shortcut with a new key.

**Comment by JorgeR81** [comment]
> In the meanwhile, another good alternative is to simplify the shortcuts like I did:
> 
> C -> Collapse
> B -> Bypass
> M -> Mute

> Obviously for the last one I had to replace Model Library shortcut with a new key.

Yes, that helps. Thanks !

---

By the way, do you use "Mute" all that often?

I just use it on the "last node" ( e.g. mute an `Image Preview` to also disable the respective `KSampler` ).
But bypass would also work in this case.

I also use "Mute" with the `Any Switch (rgthree)`, but for this I also have the `Fast Muter (rgthree)`.

**Comment by LukeG89** [comment]
> By the way, do you use "Mute" all that often?

Not that often actually, I just thought to simplify them all. But on second thought, Mute could remain the default one.

**Comment by tiptinker** [comment]
Some nodes are very small, not possible to add a button on it.

**Comment by JorgeR81** [comment]
> Some nodes are very small, not possible to add a button on it.

In think it could work like this:

- **The button would be hidden by default, in all nodes.**
We would have a right-click menu option to: `add bypass toggle` ( to all the nodes of the same type ).
I would like to bypass a "Load Lora" or "Apply ControlNet" node, but not a "Load Checkpoint" or "Load ControlNet".

- It will not be a bulky button, just one icon ( that could change shape when we click on it ).
It could be similar to the icons used in the rgthree group toggles ( see image above, in the group's top right corner )

- The collapsed nodes, could be made just a little longer to fit the icon / button.
Alternatively, the button could be hidden, if the node is collapsed. I wouldn't mind that.
